packages/proton-bridge/replace_intro.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/internal/frontend/cli/frontend.go b/internal/frontend/cli/frontend.go
```

Comment:
Packages must be as close as possible to original and contain only patches fixing compatibility with Android OS and Termux filesystem layout. Everything beyond that is called "customization" and not eligible to be part of Termux package.

Delete this patch.

Reply:
The ASCII graphics is so large that it wraps and fills all screen with garbage.